compiler/generator/rust/rust_code_container.cpp

```diff
@@ -204,13 +266,13 @@ void RustCodeContainer::produceClass()
         tab(n, *fOut);
         *fOut << "}";
         tab(n, *fOut);
-        *fOut << "fn remainder_f32(from: f32, to: f32) -> f32 {";
+        *fOut << "pub fn remainder_f32(from: f32, to: f32) -> f32 {";
```

Comment:
Why make these (`remainder_f32`, `rint_32`, `remainder_64`, `rint_f64`) public?

As far as I can see they are only supposed to be used privately so they can be kept private, no?